aimmo-game/simulation/pickups.py

```diff
@@ -29,8 +29,15 @@ def serialise(self):
 
 class HealthPickup(_Pickup):
     def __init__(self, cell, health_restored=3):
-        super(HealthPickup, self).__init__(cell)
-        self.health_restored = health_restored
+        # Round the value to the nearest integer.
+        health_restored = int(round(health_restored))
```

Comment:
So as we discussed yesterday, the user does not access this method directly so checking is not required (?). A 'generic' exception is thrown by round() and that's adequate enough.

I see where you are coming from, because we try to think like Java, but don't think it's a pythonic way of checking types for every method param.

Reply:
kool